src/web/mjs/connectors/ShueishaMangaPlus.mjs

```diff
@@ -33,31 +33,32 @@ export default class ShueishaMangaPlus extends Publus {
     }
 
     async _getMangas() {
-        let request = new Request(new URL('/api/title_list/all', this.apiURL), this.requestOptions);
+        let request = new Request(new URL('/api/title_list/allV2', this.apiURL), this.requestOptions);
         let data = await this.fetchPROTO(request, this.protoTypes, this.rootType);
-        return data.success.allTitlesView.titles.map(manga => {
+        return data.success.allTitlesViewV2.allTitlesGroup.flatMap(mangaGroup => mangaGroup.titles.map(manga => {
             return {
                 id: manga.titleId,
                 title: `${manga.name} ${this._getLanguage(manga.language)}`
```

Comment:
you should update language array 

![image](https://github.com/user-attachments/assets/67917250-c6a4-4425-950b-fece24f2ae57)

Reply:
i hate pictures of text (but i still love your PR), episode 65536:
`            0: '[en]', 1: '[es]', 2: '[fr]', 3: '[id]', 4: '[pt-br]', 5: '[ru]', 6: '[th]', 7: '[de]', 8: '[unk]', 9: '[vi]'`
PS: what is this `unk`  lang? i don't see it on shueisha manga list....

Reply:
I wasnt able to find the labge for 8. But since we have a 9 we ought to have a 8 .